### ISSUE TITLE
Put default values into host.json for discoverability

### ISSUE THREAD

**Issue by paulbatum** [Azure/azure-webjobs-sdk-script#325]
The default host.json file is empty which makes the configurable knobs difficult to discover. Better for the host to have explicit defaults in the host.json so people can discover them and modify the values.

Related: #311

**Comment by davidebbo** [comment]
Yes, that's a simple step that will go a long way to allowing people to tweak the values without having to dig things out of documentation.

**Comment by fabiocav** [comment]
@paulbatum the schema we've published since this issue was opened has helped with this, but I think it would still be helpful.

Will move this to `triaged` so we can discuss.

**Comment by paulbatum** [comment]
This is an old issue. Time has given me some perspective :) The downside of putting defaults into host.json are:
1. When a customer files an issue today, we often ask them for host.json. Today it is simple to make sense of what they give us. It will be harder if its a full file with mostly default values. Identifying something unusual will take longer.
2. Sometimes, we can change a default value in a non-breaking way, and everyone that has not overridden that setting will get the benefit of that change. If we generate a full host.json, only new apps will get the benefit.
